source/borderrouter_tasklet.c

```diff
@@ -59,6 +55,9 @@ conf_t *global_config = static_config;
 #define NR_BACKHAUL_INTERFACE_PHY_DRIVER_READY 2
 #define NR_BACKHAUL_INTERFACE_PHY_DOWN  3
 
+static mac_api_t *api;
```

Comment:
Invalid comment. Kevin told that by default they are initialized to 0.
Ignore my comment...